shtdlib.sh

```diff
@@ -216,10 +237,10 @@ function readlink_m {
 function compare_versions {
     versions=(${@})
     if [ "$(sort --help | grep version-sort)" != '' ] ; then
```

Comment:
Can we remove this if condition now since the version_sort function will do this check?

Reply:
Yes, that was half the point 👍